src/bo4e/cases.py

```diff
@@ -0,0 +1,12 @@
+from stringcase import camelcase, snakecase
+from marshmallow import pre_load, post_dump
+
+
+class JavaScriptMixin:
+    @pre_load
+    def to_snakecase(self, data, **kwargs):
+        return {snakecase(key): value for key, value in data.items()}
+
+    @post_dump
+    def to_camelcase(self, data, **kwargs):
+        return {camelcase(key): value for key, value in data.items()}
```

Comment:
Diese Datei dient dazu, dass wir in der Python-Welt immer snake_case verwenden und in dem JSON (Javascript-Welt) CamelCase verwenden. Sie wird immer vor dem laden bzw. nach dem dumpen der JSON Datei ausgeführt.

Reply:
Kannst du das als Kommentar/Docstring in die Klasse schreiben?

Reply:
done